generator/lookml.py

```diff
@@ -27,11 +27,20 @@ def _generate_views(
 
 
 def _generate_explores(
-    client, out_dir: Path, namespace: str, explores: dict, views_dir: Path
+    client,
+    out_dir: Path,
+    namespace: str,
+    explores: dict,
+    views_dir: Path,
+    v1_name: Optional[str],
```

Comment:
I regret now calling this `v1_name`. No need to change it everywhere (but feel free if you want to), however, can you add a doc line here about what it is?

Reply:
Yup. I suspect the right way of dealing with v1_name is through changes to the probeinfo service and/or mozilla-schema-generator https://github.com/mozilla/lookml-generator/pull/143/commits/24386055db4638fa73cc8ea27a4d24455923f07c